packages/web/src/react-query/token/queries.ts

```diff
@@ -0,0 +1,25 @@
+import { TokenListResponse } from "@repositories/token";
+import { UseQueryOptions, useQuery } from "@tanstack/react-query";
+import { getTokenPrices, getTokens } from "./request";
+import { QUERY_KEY } from "./types";
+
+export const useGetTokensList = (
+  option?: UseQueryOptions<TokenListResponse, Error>
+) => {
+  return useQuery<TokenListResponse, Error>({
```

Comment:
I think it would be a good idea to load the tokenRepository here.
You can use a hook called useGnoswapContext.

Reply:
fixed